validators/RequestSchema.js

```diff
@@ -11,6 +11,23 @@ const ItemSchema = FileSchema.keys({
   children: Joi.array().items(Joi.string()),
 })
 
+// Homepage
+const UpdateHomepageSchema = Joi.object({
+  content: Joi.object({
+    frontMatter: Joi.object({
+      layout: Joi.string().required(),
+      title: Joi.string().required(),
+      description: Joi.string(),
+      permalink: Joi.string().required(),
+      notification: Joi.string(),
+      image: Joi.string(),
+      sections: Joi.array().required(),
+    }).required(),
+    pageBody: Joi.string().allow(""),
```

Comment:
could we leave a comment here as to why we allow empty strings instead of making it not required? it's not entirely clear to me the reason why (maybe FE always defaults this?) and it's different from the other validation here

Reply:
Joi seems to disallow empty strings if you only specify that it is of type string, e.g. `pageBody: Joi.string()` would accept an object with no pageBody but not one with `pageBody: ''` unless explicitly allowed. I've added a comment in https://github.com/isomerpages/isomercms-backend/pull/331/commits/540bba889a9fcddc1d8c5be62901c9ea08d9df6f to explain this!